examples/website/src/shared/route/router.tsx

```diff
@@ -22,6 +29,10 @@ export function Router() {
     }
   }, [])
 
+  useEffect(() => {
+    analytics.page()
+  }, [])
```

Comment:
should probably add current to this array, so it sends the page event on every navigation